### PR TITLE
Review subresource-loading (part 1)

### PR DESCRIPTION
**Pull request by yoavweiss**
Reviewed the first ~third of the doc. Will continue later...

### CODE REVIEW

File: subresource-loading.md [hunk 1]
```diff
@@ -47,15 +47,15 @@ Existing bundlers like webpack should be able to support a mode outputting resou
 
 #### For servers
 
-- Web servers, as well as optimizing/caching intermediaries such as CDNs, should be able to implement the appropriate serving side of resource bundles within reasonable resource constraints.
-- The serving logic should be somehow "commoditized" and not require a lot of extra work to get going (unlike HTTP/2 PUSH).
-- Protocols must be stateless, with the server not having to remember what an individual client's cache contains.
-- Deployment of static content using resource bundles to servers should be possible simply by copying files over, for a server which supports resource bundles. Deployment should be stateless: the server shouldn't have to remember anything about the previous version of the site.
```
Comment (littledan) [inline subresource-loading.md:53]: Why did you delete this sentence?

**Review by littledan — APPROVED** [review]
Thanks for the edits; LGTM modulo a question

File: subresource-loading.md (hunk 1 above)
Comment (yoavweiss) [reply subresource-loading.md:53]: I folded it into the previous sentence, but that's editorial, so no strong opinions on that
